### PR TITLE
feat: created ProductComponent and fetched data from backend 

### PR DESCRIPTION
**Pull request by Shubhamkr2610**
ProductPage and ProductComponent
I have added the Product page and ProductCard component and fetched data from backend  also reafctored class to className . So ignore other files. Focus only on the ProductPage.js (in pages ) , ProductCard.jsx (in components) .

Files to check
src/components/product-card/ProductCard.jsx
src/components/product-page/ProductPage.jsx
src/pages/landingpage/Landingpage.js
PS - I refactored class to className , you can check  in 
src/components/header/header.jsx
src/components/footer/footer.jsx. 

What not to check
Responsiveness
Navbar functionalities

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import React from "react";
 import { Header, Footer } from "./components/index.js";
-import { Cart, LandingPage, Login, Products, Signup, Wishlist } from "./pages/index.js";
+import { Cart, LandingPage, Login, ProductPage, Signup, Wishlist } from "pages";
```
Comment (arjun-computer-geek) [inline src/App.js:3]: This is a good way of importing and exporting

**Review by arjun-computer-geek — APPROVED** [review]
it look good. Good code of conduct. easily readable code

**Review by sumitk-singh — APPROVED** [review]
code looks clean, good to merge.

**Comment by developer-anand** [comment]
Good to merge.

**Review by developer-anand — APPROVED** [review]
Good to merge.

**Pull request merged**